.github/workflows/translations-sync.yml

```diff
@@ -3,27 +3,27 @@
 
 # See translations-upload.yml for automation to upload our source content
 # See translations-pr-lint.yml for quality control we conduct on ingress of new translations.
-name: Crowdin Download
+name: Crowdin Translations Sync
 
 on:
-  workflow_dispatch: # Allow running when we want to, for events such as urgent translation mistakes or 100% completed languages
+  workflow_dispatch: # Manual trigger for urgent fixes
   schedule:
-    - cron: '0 5 * * 5' # At 05:00 on Fridays. This guarantees that we have the 72 hour weekend time to review translations.
+    - cron: '0 5 * * 5' # Fridays at 05:00 UTC, allowing weekend review time
 
 # Cancel any runs on the same branch
 concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
-permissions:
-  contents: read
+permissions: {}
+
+env:
+  COMMIT_MSG: 'chore: sync translations from crowdin'
```

Comment:
re-using this twice destroys our ability to see that a formatting occurred post-ingestion. This has been useful information in the past. I'd prefer if both commits here have different messages.